### PR TITLE
OK 47 fix movie tile

### PR DESCRIPTION
**Pull request by dufourmax1738**
# Jira Link:

[[Jira: ]()
](https://champlainsaintlambert.atlassian.net/jira/software/projects/OK/boards/55?selectedIssue=OK-47)
# Context

Fixed the movie tile so that it can be clicked on from every page

# Changes

Changed the movie tile href to be an absolute link instead of relative.

### CODE REVIEW

**Review by ClementCadieux — APPROVED** [review]
Simple enough fix, well done.

**Review by DuncanBH — CHANGES_REQUESTED** [review]
Pretty good change, have you checked if this issue exists anywhere else in the website?

File: components/MovieTile.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function MovieTile(props: propsType){
     return (
         <div style={{margin:"1rem"}} data-cy={`MovieTile`}>
             <Card className="bg-light-gray" style={{width:"14rem"}}>
-                <Link href={"./movies/" + props.movie.id}>
+                <Link href={"/movies/" + props.movie.id}>
```
[the remark this answers is not in the document]
Comment (dufourmax1738) [reply components/MovieTile.tsx:13]: I could but it doesn't seem necessary so I will not be doing that change

**Comment by dufourmax1738** [comment]
> 

I have and did not see any other issue like this

**Review by DuncanBH — APPROVED** [review]

**Pull request merged**